src/web/pages/alerts/emailmethodpart.jsx

```diff
@@ -199,16 +199,20 @@ const EmailMethodPart = ({
                       onChange={handleReportFormatIdChange}
                     />
                   )}
-
-                  <label htmlFor="report-config-select">Report Config</label>
-                  <Select
-                    disabled={notice !== EMAIL_NOTICE_INCLUDE}
-                    name={prefix + 'notice_report_config'}
-                    id="report-config-select"
-                    value={reportConfigIdInState}
-                    items={reportConfigItems}
-                    onChange={handleReportConfigIdChange}
-                  />
+                  {
+                    capabilities.mayOp("get_report_configs") &&
```

Comment:
I think it is better to replace passing `capabilities` as a prop with the `useCapabilities` hook. 

The export will need an update to remove `withCapabilities`